frontend/css/playlists.less

```diff
@@ -40,34 +89,86 @@
           margin: 0;
         }
       }
+    }
+  }
 
-      /* Pure CSS multiline ellipsis by Sagi Shrieber:
-			   http://hackingui.com/a-pure-css-solution-for-multiline-text-truncation/
-			*/
-      .description {
-        overflow: hidden;
-        position: relative;
-        line-height: 1.2em;
-        max-height: @descriptionLines * 1.2em;
-        text-align: justify;
-        padding-right: 2em;
-        &:before {
-          content: "...";
-          position: absolute;
-          right: 1em;
-          bottom: 0;
+  .playlist-card-list-view {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid #e2e8f0;
+    padding: 1rem;
+    margin: 0.5rem 0;
+    min-height: 8rem;
+    border-radius: 10px;
+    gap: 3rem;
+
+    @media (max-width: 640px) {
+      gap: 0;
+    }
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    .playlist-card-container {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      gap: 0.8rem;
+
+      .playlist-info {
+        display: flex;
+        gap: 2rem;
+
+        .playlist-index {
+          min-width: 2rem;
+          text-align: right;
+          font-weight: bold;
+          margin-top: 0.5rem;
+        }
+      }
+    }
+
+    .playlist-info-content {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: 1rem;
+
+      @media (min-width: 640px) {
+        margin-bottom: 0;
+      }
+
+      .playlist-title {
+        font-size: 1.75rem;
+        margin-bottom: 0.5rem;
+        a {
+          font-weight: 500;
+          color: #353070;
+          text-decoration: none;
         }
-        &:after {
-          content: "";
-          position: absolute;
-          right: 0;
-          width: 1em;
-          height: 1em;
-          margin-top: 0.2em;
-          background-color: inherit;
+      }
+    }
+
+    .playlist-more-info {
+      align-self: center;
```

Comment:
Add this to prevent shrinking when there is a description:
```suggestion
      align-self: center;
      flex-shrink: 0;
```

Before:
![image](https://github.com/user-attachments/assets/ba3e5df8-47a6-4c96-b711-a742d5978e50)

After:
![image](https://github.com/user-attachments/assets/bd56cfc0-a113-4fd2-a06c-01faace512ef)